deploy/sbin/litcal-fpm-reload.sh

```
@@ -0,0 +1,69 @@
+  # and compare the counter, so a crash-loop is reported here rather than
+  # discovered days later.
+  sleep 8
+  restarts_after="$(systemctl show "$WS_UNIT" -p NRestarts --value 2>/dev/null || echo 0)"
+  if [ "$restarts_after" -gt "$restarts_before" ]; then
+    logger -t litcal-fpm-reload "${WS_UNIT} is CRASH-LOOPING after deploy (NRestarts ${restarts_before}->${restarts_after}); check: journalctl -u ${WS_UNIT} -n 50"
+    exit 1
+  fi
+  logger -t litcal-fpm-reload "${WS_UNIT} restarted and stable"
+fi
+
+rm -f $SENTINELS
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)